src/ResourceManager/Network/Commands.Network/ChangeLog.md

```diff
@@ -28,6 +28,7 @@
 * Added cmdlet Get-AzureRmFirewallFqdnTag
 * Added Exclusion list and Global config functionality for application gateway WAF, new cmdlets added
 * Fixed issue with RouteFilter not being set if it was passed to ExpressRouteCircuit Peering as resource
+* Added cmdlet New-AzureRmApplicationGatewayCustomError, Add-AzureRmApplicationGatewayCustomError, Get-AzureRmApplicationGatewayCustomError, Set-AzureRmApplicationGatewayCustomError, Remove-AzureRmApplicationGatewayCustomError, Add-AzureRmApplicationGatewayHttpListenerCustomError, Get-AzureRmApplicationGatewayHttpListenerCustomError, Set-AzureRmApplicationGatewayHttpListenerCustomError, Remove-AzureRmApplicationGatewayHttpListenerCustomError
```

Comment:
@KaiChiLin  move this under the `Current Release` header.. this might have gotten messed up in the merge you did form the upstream branch